requirements/base.txt

```diff
@@ -49,7 +49,7 @@ django-ipware==2.0.2
 geoip2==2.8.0
 django-silk==2.0.0
 django-extensions==2.2.1
-ipdb==0.11
+ipdb==0.13.9
```

Comment:
why are we bumping this here ? Could this be done in it's own PR ?

Reply:
You actually already merged this in another PR: https://github.com/gitcoinco/web/pull/9443 :-) But I updated the base so we're good now.